xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/service/BittrexTradeService.java

```diff
@@ -47,6 +47,14 @@ public String placeMarketOrder(MarketOrder marketOrder) throws IOException {
     }
   }
 
+  public LimitOrder placeMarketOrder2(MarketOrder marketOrder) throws IOException {
```

Comment:
placing a market order results in a limit order ?

Reply:
yes, a market order is a special case of limit order

Reply:
no, its not, it has no limit price